### PR TITLE
Added hotkey EDIT: (ctrl+H) to like/favorite track. closes #716

### LINKED ISSUE #716: [FEATURE SUGGESTION] Hotkey to like a song
A keyboard shortcut or a hotkey of some kind that would allow the user to like the current song they're listening to.

### PR DESCRIPTION
**Pull request by mathiash98**
As requested by another user. issue #716 
Done using triggerHandler on ".player_favorite"

### CODE REVIEW

**Comment by zestyr** [comment]
Currently, the shortcut CTRL+L "locks" the track, you may wanna look into that

**Comment by weblancaster** [comment]
@zestyr is right @mathiash98 you will need to change that

**Comment by zestyr** [comment]
I'd suggest Ctrl+H, since it isn't taken, and it's H as in Heart

**Comment by mathiash98** [comment]
@zestyr @weblancaster edited to ctrl+H now

File: app/public/js/player/playerCtrl.js [hunk 1]
```diff
@@ -289,6 +289,16 @@ app.controller('PlayerCtrl', function (
     });
 
     hotkeys.add({
+        combo: ['command+h', 'ctrl+h'],
+        description: 'Heart track',
+        callback: function() {
+          $timeout(function() {
```
Comment (weblancaster) [inline app/public/js/player/playerCtrl.js]: why the `$timeout` here?

Comment (mathiash98) [inline app/public/js/player/playerCtrl.js]: I have no idea, I just copied your code from another function. @weblancaster

Comment (weblancaster) [inline app/public/js/player/playerCtrl.js]: Hm I checked not sure why we have that timeout over there.. please remove and test again if works just leave the $timeout out and I will merge.. thanks

**Comment by 1percent** [comment]
@mathiash98 agreed, this is implementation is very important.

**Pull request merged**